pkg/apis/aquasecurity/v1alpha1/vulnerability_types.go

```diff
@@ -78,12 +79,15 @@ type Vulnerability struct {
 	Severity    Severity `json:"severity"`
 	Title       string   `json:"title"`
 	Description string   `json:"description,omitempty"`
+	CVSSSource  string   `json:"cvsssource,omitempty"`
 	PrimaryLink string   `json:"primaryLink,omitempty"`
 	//+optional
 	Links []string `json:"links"`
 	Score *float64 `json:"score,omitempty"`
 	//+optional
 	Target string `json:"target"`
+	//+optional
+	CVSS types.VendorCVSS `json:"cvss,omitempty"`
```

Comment:
Please check an optional validation annotation is required `// +kubebuilder:validation:Optional`
[kube-builder reference](https://book.kubebuilder.io/reference/markers/crd-validation.html)

Reply:
Ok!